bns/getKoboData.js

```diff
@@ -1,45 +1,45 @@
+import { http } from '@openfn/language-googlesheets';
 //== Job to be used for fetching data from Kobo on repeated, timer basis  ==//
 // This can be run on-demand at any time by clicking "run" //
+getValues(
+  '1s7K3kxzm5AlpwiALattyc7D9_aIyqWmo2ubcQIUlqlY',
+  'sheetsList of Kobo forms!A:L',
+  state => {
+    const [headers, ...values] = state.data.values;
+
+    const filteredSheetsData = values.filter(item => {
+      if (
+        item.includes('TRUE') &&
+        item.includes('bns_survey', 'nrgt_current')
+      ) {
+        return item;
```

Comment:
You don't have to say `return item`. The callback function that `array.filter(cbFn)` takes, should return a true value to keep the element in the resulting array. [See MDN  docs on Array.filter() to learn more](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)